### PR TITLE
EM Automation - Migrating Test Cases from SM Test Automation Framework

### PR DESCRIPTION
**Pull request by iamdez99**
EM Automation - Migrating Test Cases from SM Test Automation Framework

### CODE REVIEW

File: src/test/java/GrantsApps/GrantsRunners/EnterpriseMaintenance/RunEMProgressionTest.java
Comment (georgeb79) [inline src/test/java/GrantsApps/GrantsRunners/EnterpriseMaintenance/RunEMProgressionTest.java:1]: Missing monochrome and strict from this runner

File: src/test/java/GrantsApps/EM/StepImplementation/EMStepsImplementation.java [hunk 1]
```diff
@@ -291,4 +291,17 @@ public static void user_can_verify_that_return_to_manage_i2e_users_hyperlink_dir
...
+        MiscUtils.sleep(2000);
+        JavascriptUtils.clickByJS(manageI2EUsersPage.changeuserDr);
+        MiscUtils.sleep(4000);
+        CommonUtils.clickOnElement(manageI2EUsersPage.changeuserField);
+        MiscUtils.sleep(2000);
+        CommonUtils.sendKeys(manageI2EUsersPage.changeuserFieldInput, "Li, Bin");
+        MiscUtils.sleep(2000);
+        CommonUtils.waitForClickability(manageI2EUsersPage.liBinOption);
+        CommonUtils.clickOnElement(manageI2EUsersPage.liBinOption);
+        MiscUtils.sleep(2000);
```
Comment (hamidnesar) [inline src/test/java/GrantsApps/EM/StepImplementation/EMStepsImplementation.java:305]: Are these going to be replaced with "wait for element clickability"

**Review by andreyeve — APPROVED** [review]

**Review by georgeb79 — APPROVED** [review]

**Review by hamidnesar — APPROVED** [review]

File: src/test/java/GrantsApps/EM/Features/EMFlow.feature
Comment (puzikovam2) [inline src/test/java/GrantsApps/EM/Features/EMFlow.feature:1]: Please delete extra line 23

**Review by puzikovam2 — APPROVED** [review]
approved by @puzikovam2

File: src/test/java/GrantsApps/EM/StepImplementation/EMStepsImplementation.java (hunk 1 above)
Comment (iamdez99) [reply src/test/java/GrantsApps/EM/StepImplementation/EMStepsImplementation.java:306]: yes

**Pull request merged**